frontend/packages/operator-lifecycle-manager/src/components/operand/const.ts

```diff
@@ -1,10 +1,13 @@
+import i18next from 'i18next';
 import { JSONSchema6 } from 'json-schema';
 import { JSONSchemaType } from '@console/shared/src/components/dynamic-form';
 
-export const YAML_HELP_TEXT =
-  'Create by manually entering YAML or JSON definitions, or by dragging and dropping a file into the editor.';
-export const FORM_HELP_TEXT =
-  'Create by completing the form. Default values may be provided by the Operator authors.';
+export const YAML_HELP_TEXT = i18next.t(
```

Comment:
These won't be reactive. We should probably remove these as constants and just inline them in the component. (If they're used in more than one place, we can create custom components that are just for the messages.)